themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -183,6 +254,24 @@
   z-index: 3;
 }
 
+.CDB-infowindow.has-header-image .CDB-hook-image {
+  position: absolute;
+  left: 24px;
+  z-index: 10;
+}
+
+.CDB-infowindow.has-header-image .CDB-hook-image:before {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 0;
+  height: 0;
+  border-right: 24px solid transparent;
```

Comment:
Properties should be ordered position, top, left, width, height, border-top, border-right, content, z-index